htdocs/admin/pdf.php

```diff
@@ -558,6 +566,27 @@
 print $form->selectarray('MAIN_GENERATE_DOCUMENTS_SHOW_FOOT_DETAILS', $arraydetailsforpdffoot, (!empty($conf->global->MAIN_GENERATE_DOCUMENTS_SHOW_FOOT_DETAILS) ? $conf->global->MAIN_GENERATE_DOCUMENTS_SHOW_FOOT_DETAILS : 0));
 print '</td></tr>';
 
+// Show alias in thirdparty name
+
+print '<tr class="oddeven"><td>'.$langs->trans("PDF_INCLUDE_ALIAS_IN_THIRDPARTY_NAME").'</td><td>';
```

Comment:
The 2 options can be merged into one.
Instead of PDF_INCLUDE_ALIAS_IN_THIRDPARTY_NAME + PDF_INCLUDE_ALIAS_IN_THIRDPARTY_NAME we can have
PDF_THIRDPARTY_NAME_IS   with value   'thirdparty'  (current behaviour)   or   'alias+thirdparty'    or 'thirdparty+alias'

Reply:
@eldy I your propose this changemente